### DISCUSSION TITLE
How can I start a server that uses one gguf for the model, and a separate gguf for the embeddins target?

### DISCUSSION THREAD

**Discussion by MrDowntempo** [ggml-org/llama.cpp#7721] [Q&A] [answered]
running a server of a gguf has been pretty simple to setup. I know that embeddings are supported, but the --embeddings flag doesn't seem to accuept a separate gguf for embeddings specifically. If I want to use a llama3 model as the main model, but nomic-embed for the embeddings target, how is that accomplished?

**Comment by ggerganov** [comment, marked as answer]
Currently this is not supported. You can try to start a second instance of `server`

**Reply by MrDowntempo** [reply]
Thank you, that's what I expected but I couldn't find it explicitly stated.
